x/mongo/driver/topology/pool.go

```diff
@@ -343,6 +340,10 @@ func (p *pool) get(ctx context.Context) (*connection, error) {
 				Reason:  event.ReasonTimedOut,
 			})
 		}
+		ErrWaitQueueTimeout := WaitQueueTimeoutError{
+			Message: "timed out while checking out a connection from connection pool",
+			Wrapped: ctx.Err(),
+		}
```

Comment:
To check my understanding, if Go 1.13 was our minimum (and [the %w format verb was available](https://blog.golang.org/go1.13-errors)), would this be roughly equivalent to:

```
return errors.Errrof ("timed out while checking out a connection from connection pool: %w", ctx.Err())
```

Reply:
From a cursory readthrough of the documentation, it looks like that would be functionally identical.